features/step_definitions/artifacts.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const { Before, Then, When } = require('cucumber');
+const request = require('screwdriver-request');
+
+const TIMEOUT = 240 * 1000;
+
+Before('@artifacts', function hook() {
+    this.repoName = 'functional-artifacts';
+
+    // Reset shared information
+    this.buildId = null;
+    this.pipelineId = null;
+    this.eventId = null;
+    this.jwt = null;
+});
+
+When(/^the "ziped" job is started$/, { timeout: TIMEOUT }, function step() {
+    return request({
+        url: `${this.instance}/${this.namespace}/events`,
+        method: 'POST',
+        json: {
+            pipelineId: this.pipelineId,
+            startFrom: 'ziped'
+        },
+        context: {
+            token: this.jwt
+        }
+    })
+        .then((resp) => {
+            Assert.equal(resp.statusCode, 201);
+            this.eventId = resp.body.id;
+        })
+        .then(() =>
+            request({
+                url: `${this.instance}/${this.namespace}/events/${this.eventId}/builds`,
+                method: 'GET',
+                context: {
+                    token: this.jwt
+                }
+            })
+        )
+        .then((resp) => {
+            Assert.equal(resp.statusCode, 200);
+            this.buildId = resp.body[0].id;
+        });
+});
+
+Then(/^the "ziped" build succeeds$/, { timeout: TIMEOUT }, function step() {
+    return this.waitForBuild(this.buildId).then((resp) => {
+        Assert.equal(resp.body.status, 'SUCCESS');
+        Assert.equal(resp.statusCode, 200);
+    });
+});
+
+Then(
+    /^artifacts were found in the build with the same event ID as the successful main job$/,
```

Comment:
Why event ID? The actual codes seem not to check the event ID.

Reply:
It was a leftover from when I ported it over from `event.js`.
I haven't checked the event id, so I'll fix the wording.